README.md

```diff
@@ -219,16 +219,15 @@ expect(
 ## `should` syntax
 
 In addition to the `expect` syntax, rspec-expectations continues to support the
-`should` syntax:
+non-monkey patching `should` syntax:
```

Comment:
I think we should remove this syntax as well, it's confusing to keep it when we are dropping the rest of it (I know it's not monkey patching) the rspec-should can add this and optionally monkey patch.

Reply:
Sounds good, it's even better than moving non-MP `should` over to `rspec-expectations` I proposed [here](https://github.com/rspec/rspec-core/pull/2800#discussion_r545910120). 👍